recipes/refnx/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{%set name = "refnx" %}
+{%set version = "0.0.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/refnx/refnx.git
+  git_rev: 404eceaff24d6b3ae116851dba25beb8069b0092
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
It configures the compilers used. Mainly this is of value on Mac particularly with C++. It works by sourcing some environment variables before the build. Please see the [feedstock]( https://github.com/conda-forge/toolchain-feedstock ) for more details.

Yeah, `simplejson` probably should have it as it is being compiled.

Yes, it probably should be in the FAQ.

Please feel free to open a PR for these.